### PR TITLE
Eke bb/allow access to response headers

### PR DESCRIPTION
**Pull request by eke-bb**
Related to https://github.com/bluebottlecoffee/bluebottle/pull/7971

From the email from Shipwire:
> He also recommends to start logging the time (milliseconds) that Blue Bottles server waits for responses on specific requests, along with these HTTP headers that our rate API returns: X-Request-Id (our unique id for the request, which makes it easy for us to look up the request in our logging), X-Process-Time-Seconds (how long the request took to process on our end), and then compare the times to see how long the message took to make it through the internet and their server provider's network.

So this exposes the headers on the `Response` class so we can access the headers mentioned above

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.4.1
+2.1.2
```
Comment (austinwu) [inline .ruby-version]: ???

File: lib/shipwire/orders.rb [hunk 2]
```diff
@@ -24,18 +24,10 @@ def holds(id, params = {})
       request(:get, "orders/#{id}/holds", params: params)
     end
 
-    def release(id)
```
Comment (austinwu) [inline lib/shipwire/orders.rb]: Also ???

Read the commit message, but didn't follow still

Comment (eke-bb) [reply lib/shipwire/orders.rb]: ok, so now looking back at this i realize what's going on. I had originally accidentally opened a PR against https://github.com/billr578/shipwire which is where all the differences were coming from. I'll need to revert those commits. thanks for catching this

File: .ruby-version (hunk 1 above)
Comment (eke-bb) [reply .ruby-version]: i like a nice vintage 😎

**Review by austinwu — APPROVED** [review]

**Pull request merged**